website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -80,6 +80,51 @@ Use this variant only when you need to put interactive elements such as a button
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =800x*)
 
+## Sizes
+
+Our Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium`, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, so AccordionItem sizes are not interchangeable.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+Our Accordion has two container types: `card` and `flush`.
+
+### Card
+
+The `card` variant is the default and recommended type, creating visual separation between Accordion sections and other UI elements. It helps organize content blocks and distinguish between sections within Tabs. Unlike `Cards`/`Tiles` or `Tables`, the `card` variant groups multiple expandable sections for flexible content management.
+
+!!! Do
+
+![image of DO example for the card variant](/assets/components/accordion/accordion-card-type-example.png)
+!!!
+
+### Flush
+
+Use the `flush` variant for tight spaces. For example, use it within other containers like Cards and Flyouts, or in sidebars where space is limited.
+
+!!! Do
+
+![image of DO example for the Flush variant](/assets/components/accordion/accordion-flush-example.png)
+!!!
+
+### Static AccordionItems
+
+The `isStatic` property allows an AccordionItem to be opened or closed by default, but not toggleable between its set `isOpen` property. This is useful when needing to represent a piece of information that is relevant to your Accordion content, but highlighting the `isStatic` version as the exception within the group.
```

Comment:
"as the exception within the group" doesn't make a ton of sense to me, is there a better way to say this?

```suggestion
The `isStatic` property allows an AccordionItem to be opened or closed by default, but not toggleable between its set `isOpen` property. This is useful when needing to represent a piece of information that is relevant to the Accordion content, but highlighting the `isStatic` version as the exception within the group.
```

Reply:
I made this change in the PR and will push it up, but I wanted to highlight it here as well. I went ahead and wordsmith this a bit. 

The `isStatic` property in an AccordionItem keeps it fixed in either an open or closed state based on the `isOpen` property. This means users can't toggle it open or closed. It's useful when you want to show important information in your Accordion that should always stay open or closed, making it stand out as different from the other items.